db/migrate/20160901013042_change_column_type.rb

```diff
@@ -0,0 +1,14 @@
+class ChangeColumnType < ActiveRecord::Migration
+  def change
+    connection.execute(%q{
+        alter table shares
+        alter column lat
+        type float using lat::float
+    })
+    connection.execute(%q{
+        alter table shares
+        alter column lon
+        type float using lon::float
+    })
```

Comment:
I originally followed this post, https://codequizzes.wordpress.com/2013/06/06/rails-migrations-to-add-a-column-and-change-column-type/, when I tried change_column, I was getting the following error "PG::Error: ERROR:  column "lat" cannot be cast to type integer".  I tried Brian's answer from [this stack overflow post ](http://stackoverflow.com/questions/17075173/rails-migrations-change-column-with-type-conversion), but it still didn't work. Then I found this stack overflow post and it worked http://stackoverflow.com/questions/10690289/rails-gmaps4rails-gem-on-postgres. Is there a more elegant way to get it working is I see an error like this? ""PG::Error: ERROR:  column "lat" cannot be cast to type integer"."